README.md

```diff
@@ -1,47 +1,48 @@
 # Adyen dotnet API Library
 [![nuget](https://img.shields.io/nuget/v/adyen.svg)](https://www.nuget.org/packages/adyen/) [![nuget](https://img.shields.io/nuget/dt/adyen.svg)](https://www.nuget.org/packages/adyen/) ![.NET Core](https://github.com/Adyen/adyen-dotnet-api-library/workflows/.NET%20Core/badge.svg)
 
-The Adyen API Library for .net framework enables you to work with Adyen APIs, Hosted Payment Pages and Terminal API with any .net application.
-
-The Library supports all APIs under the following services:
-* [x] bin lookup
-* [x] checkout
-* [x] checkout utility
-* [x] payments
-* [x] payout
-* [x] modifications
-* [x] recurring
-* [x] notifications
-* [x] marketpay account
-* [x] marketpay fund
-* [x] Terminal API (Local and Cloud based)
-
-## Requirements
-* Adyen API Library supports .net standard 2.0
-* In order for Adyen API Library to support local terminal api certificate validation the application should be set to .net core 2.1 and above or .net framework 4.6.1 and above
+This is the officially supported dotnet library for using Adyen's APIs.
+
+## Integration
+The library supports all APIs under the following services:
+
+* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/overview): Our latest integration for accepting online payments. Current supported version: **v67**
+* [Payments API](https://docs.adyen.com/api-explorer/#/Payment/v51/overview): Our classic integration for online payments. Current supported version: **v51**
+* [Recurring API](https://docs.adyen.com/api-explorer/#/Recurring/v25/overview): Endpoints for managing saved payment details. Current supported version: **v49**
+* [Payouts API](https://docs.adyen.com/api-explorer/#/Payout/v51/overview): Endpoints for sending funds to your customers. Current supported version: **v51**
+* [Adyen BinLookup API](https://docs.adyen.com/api-explorer/#/BinLookup/v50/overview): Endpoints for retrieving information, such as cost estimates, and 3D Secure supported version based on a given BIN. Current supported version: **v50**
+* [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v1/post/originKeys): This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version: **v1**
```

Comment:
same as https://github.com/Adyen/adyen-python-api-library/pull/140/files#r621982909

Reply:
Done set to v67